docs/book/cookbook/bootstrap.md

```diff
@@ -0,0 +1,46 @@
+# Use Alerts from Bootstrap
+
+To use the syntax of the [alerts from the Bootstrap CSS framework](https://getbootstrap.com/docs/5.2/components/alerts/) for the output of the view helper, the configuration of the message format must be adjusted.
+
+Add the following lines to the local or global configuration file, e.g. `config/autoload/global.config.php`:
```

Comment:
I'd start this sentence with "To do that, add" as it is an instruction that directly leads on from the previous paragraph.